### ISSUE TITLE
wrong PARG gene transcript from ENSEMBL

### ISSUE THREAD

**Issue by jjgao** [genome-nexus/genome-nexus#193]
Reported by a user:

"
The PARG gene transcript used as a reference for cBioportal is from ENSEMBL and encodes a 491 amino acid protein.

It should be noted that is ENSEMBL sequence is incorrect. Full length PARG is 794 amino acids (GenBank: AAH52966.1) and the nucleotide sequence of the mRNA is 4,312 bp (NM_003631.4    GI: 1198333560).

  The ENSEMBL sequence is simply incorrect. The GENEbank sequence should be used as a reference.
"

https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/cbioportal/xF9pj-sWItw/SO3KTsXqBAAJ

**Comment by inodb** [comment]
@jjgao this is a tricky one. On grch37.ensembl.org it's only the 491aa one. On ensembl.org there are multiple, all different from genbank length of 794 aa. The longest being 976aa. So either we can check with the user if the 976aa protein is good as well instead of the genbank one. Then we can set up an instance to support the latest biomart dump. I don't think allowing to select genbank transcripts makes sense since the vep results won't make sense unless there's actually a matching protein in ensembl.

**Comment by jjgao** [comment]
@inodb agreed. We should not select genbank transcripts. The uniprot entry (http://www.uniprot.org/uniprot/Q86W56) has 976 AAs too. But how can we support it if it's not in grch37?

**Comment by inodb** [comment]
@jjgao I can email the user that we currently only support grch37 ensembl transcripts (biomart v91) and that we are planning grch38 support but that the transcript there is different as well. Does that make sense?

**Comment by jjgao** [comment]
@inodb yes, thanks.
